Sources/Model/TransferMethod/HyperwalletTransferMethodQueryParam.swift

```diff
@@ -56,12 +58,33 @@ public class HyperwalletTransferMethodQueryParam: QueryParam {
         case descendantStatus = "-status"
     }
 
+    /// Representation of the transfer method type
+    public enum QueryType: String {
+        /// The Bank account type
+        case bankAccount = "BANK_ACCOUNT"
```

Comment:
Use HyperwalletTransferMethod.TransferMethodType.bankAccount.rawvalue instead of hard coding values here.